combat-trainer.lic

```diff
@@ -873,7 +872,10 @@ class LootProcess
   end
 
   def check_skinning(mob_noun, game_state)
+    already_arranged = false
     if @dissect && game_state.dissectable?(mob_noun)
+      arrange_mob(mob_noun, game_state) if @always_arrange
```

Comment:
I think the initialization of `@always_arrange` was accidentally removed?

Reply:
It was, re-added, renamed.